Managers/DatabaseManager.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DatabaseManager.swift
+//  Shapify
+//
+//  Created by Alex on 24/01/2023.
+//
+
+import Foundation
+import FirebaseDatabase
+
+class DatabaseManager {
+    
+    static let shared = DatabaseManager()
+    
+    private let db = Database.database().reference()
+    
+    private init(){}
+    
+    private static func safeEmail(emailAddress: String) -> String {
```

Comment:
Shouldn't this be called `saveEmail` ?

Reply:
no, because it does not save an email, but returns a safe email. I can rename it to getSafeEmail

Reply:
Then there is no need to change it :)